js/src/personal/haoking/RightAboutMe.tsx

```diff
@@ -14,6 +15,7 @@ export function RightAboutMe() {
       ml={90}
       mt={45}
       maw={400}
+      className={styles.animate_fade_in_delay}
```

Comment:
nit: I feel like it might be good practice to keep className as the first prop, but I haven't used it enough to have a firm opinion.